climpred/metrics.py

```diff
@@ -1,62 +1,11 @@
-import numpy as np
 import types
-from xskillscore import mae as _mae
-from xskillscore import mse as _mse
-from xskillscore import pearson_r as _pearson_r
-from xskillscore import rmse as _rmse
-
-
-def _control_for_reference_period(control, reference_period='MK',
```

Comment:
that can be done before. dont care for this as the bootstrapping treats every metric equally.